content/webapp/pages/stories.tsx

```diff
@@ -99,17 +99,30 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     // would it be faster to skip all the fetchLinks?  Is that possible?
     const storiesPagePromise = fetchPage(client, prismicPageIds.stories);
 
-    const seriesPromise = fetchSeriesById(client, featuredStoriesSeriesId);
+    const featuredSeriesArticlesQueryPromise = fetchArticles(client, {
```

Comment:
The reason I got this wrong is that the old code went:

```
const seriesDocument = getSeries()
const articlesDocument = getArticles()
const series = parseArticlesAndSeries(seriesDocument, articlesDocument).series
```

And I hadn't spotted that the articles got copied onto the series half of that result. More evidence that the transformation method for articles/series needs refactoring.